src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st

```diff
@@ -0,0 +1,54 @@
+"
+Description
+--------------------
+
+I am a command available in the class context menu of Calypso. When activated I'll jump on the test class corresponding to the class. 
+
+If the class does not exists I will create it in the corresponding package.
+	
+Internal Representation and Key Implementation Points.
+--------------------
+
+    Instance Variables
+	testClass:		<aClass>		Test class corresponding to the class I was activated with
+"
+Class {
+	#name : #ClyJumpToTestClassCommand,
+	#superclass : #SycSingleClassCommand,
+	#traits : 'TClyGenerateTestClass',
```

Comment:
There was one requirement for Calypso to work in system without traits. If you will look at SUnit plugin dependencies you will not see traits.
I am sure this trait can be extracted to object.

Reply:
Hi,

Can you explain the necessity of this requirement? I thought that Calypso was not planned for port to other languages and since it's a development tool I guess it's not to have minimal images without traits.

Reply:
After you wrote it I had doubts. Maybe I overthought it in the time of removing traits from kernel. It influenced this kind of things in my head.

Reply:
So, we are fine with Traits?

Reply:
I think so.